### DISCUSSION TITLE
Paste into commit body

### DISCUSSION THREAD

**Discussion by Sami6720** [jesseduffield/lazygit#3465] [Q&A]
I often find myself wanting to paste stuff into the body of a commit. However, couldn't find a way to do it in the window that pops up when you press `c`. Is there a way to do it?

**Comment by mark2185** [comment]
Regular paste doesn't work? E.g. `ctrl+shift+v` on linux or `cmd+v` on mac?

**Reply by Sami6720** [reply]
I have a Windows machine. It seems like <C-S-v> doesn't work for it.
